modules/model/shared/src/main/scala/observe/model/ObservationProgress.scala

```diff
@@ -4,42 +4,25 @@
 package observe.model
 
 import cats.Eq
-import cats.derived.*
-import cats.syntax.all.*
-import lucuma.core.model.Observation
-import lucuma.core.model.sequence.Step
-import lucuma.core.util.Enumerated
-import lucuma.core.util.TimeSpan
-import monocle.Prism
-import monocle.macros.GenPrism
+import io.circe.Decoder
+import io.circe.Encoder
+import io.circe.JsonObject
+import io.circe.syntax.*
 
-enum ObservationProgress(val isNs: Boolean) derives Eq:
-  def obsId: Observation.Id
-  def stepId: Step.Id
-  def total: TimeSpan
-  def remaining: TimeSpan
-  def stage: ObserveStage
-
-  case Regular(
-    obsId:     Observation.Id,
-    stepId:    Step.Id,
-    total:     TimeSpan,
-    remaining: TimeSpan,
-    stage:     ObserveStage
-  ) extends ObservationProgress(false)
-
-  case NodAndShuffle(
-    obsId:     Observation.Id,
-    stepId:    Step.Id,
-    total:     TimeSpan,
-    remaining: TimeSpan,
-    stage:     ObserveStage,
-    sub:       NsSubexposure
-  ) extends ObservationProgress(true)
+case class ObservationProgress(obsId: Observation.Id, stepProgress: StepProgress):
+  export stepProgress.*
```

Comment:
What does this do?

Reply:
Allows direct access to `StepProgress` members. I thought this might be convenient since I split `ObservationProgress` into 2 classes.